src/main/java/com/microsoft/azure/servicebus/jms/AadAuthentication.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.jms;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.credential.TokenRequestContext;
+
+public class AadAuthentication {
```

Comment:
Not sure if there could be only AAD tokens but I think it is better to use ServiceBusAuthentication or ServiceBusCredential name.